measures/2019/measures-data.json

```diff
@@ -46928,7 +46928,7 @@
   {
     "measureId": "ABFM8",
     "title": "Measuring the Value-Functions of Primary Care: Provider Level Continuity Measure",
-    "description": "Bice-Boxerman Continuity of Care Primary Care Physician Measure (BB-COC-PC).  \n\nAt a patient-level, BB-COC is a measure that considers the dispersion of primary care visits across providers, such that patients with higher scores have most of their primary care visits to the same provider or a small number of providers while those lower scores see a larger number providers.  Formally, an individual BB-COC score is calculated as follows:\n\nBB-COC=(·_(i=1)^k n_i^2 -N)/(N(N-1))    (1)\n\nwhere k is the number of providers, n_i is the number of visits to provider i, and N is the total number of visits.  (Note that it is necessary that the patient has at least two visits.)\nWe will calculate the physician-level continuity measure for all patients as follows: \n  \nBB-COC-PC=(·_1^k _((BB-COC)(n_k ))_)/(N*(n_k))  (2)\n\nWhere BB-COC is the individual patient continuity score, n is number of total primary care visits for patient k during the study period, and N is the total number of patients seen by the physicians during the study period.  This approach gives greater weight to patients with more visits.",
+    "description": "Bice-Boxerman Continuity of Care Primary Care Physician Measure (BB-COC-PC).  \n\nAt a patient-level, BB-COC is a measure that considers the dispersion of primary care visits across providers, such that patients with higher scores have most of their primary care visits to the same provider or a small number of providers while those lower scores see a larger number providers.  Formally, an individual BB-COC score is calculated as follows:\n\nBB-COC=(∑_(i=1)^k n_i^2 -N)/(N(N-1))    (1)\n\nwhere k is the number of providers, n_i is the number of visits to provider i, and N is the total number of visits.  (Note that it is necessary that the patient has at least two visits.)\nWe will calculate the physician-level continuity measure for all patients as follows: \n  \nBB-COC-PC=(∑_1^k _((BB-COC)(n_k ))_)/(N*(n_k))  (2)\n\nWhere BB-COC is the individual patient continuity score, n is number of total primary care visits for patient k during the study period, and N is the total number of patients seen by the physicians during the study period.  This approach gives greater weight to patients with more visits.",
     "nqfId": null,
     "nationalQualityStrategyDomain": "Communication and Care Coordination",
     "measureType": "structure",
```

Comment:
This looks a bit strange, `nBB-COC-PC=(∑_1^k _((BB-COC)(n_k ))_)`, think it's what is intended?

Reply:
I think so, looks like a sigma notation math formula

Reply:
I think the \n\n is the paragraph breaks. also this comment is on the old version so not sure what specifically I'm supposed to review? the green side looks like the right formula except for the \n\n but I'm not sure how to get rid of those except to take out all of the extra lines in the excel?